index.html

```diff
@@ -41,22 +51,22 @@ <h2>Save the Date</h2>
         <h3>2022/09/29(Thurs) - 30th(Fri)</h3>
       </div>
     </section>
-</div>
     <section class="events-section">
```

Comment:
The spacing of your main program section does not follow the [template design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [ ]  Contents in this section should have appropriate padding all around them. (See the indication from the attached screenshot)
- [ ] The _link_ below the content of this section, _see the whole program_ should be in **uppercase**.
**Your main program section:**
![image](https://user-images.githubusercontent.com/12745474/169594860-43c5cab5-cf8a-45df-a516-4398acf70b0c.png)


**The original main program section:**
![image](https://user-images.githubusercontent.com/12745474/169595746-df4272f5-3f63-4487-9406-4c406e41eb72.png)